### ISSUE TITLE
bug: parse address list from prompt

### ISSUE THREAD

**Issue by pazz** [pazz/alot#889]
When I prompt for a list of recipients in envelope mode and type in 

>  "X X" x@x.com, "Y Y" y@y.com

The `To` header gets assigned the value

>  "X X x@x.com, Y Y" y@y.com

which is malformed and leads to mail delivery errors.

**Comment by pazz** [comment]
note to self: there should be a test for this

**Comment by lucc** [comment]
I can only reproduce this half way. I use `msmtp --file /home/luc/.config/msmtp/msmtprc --read-recipients --read-envelope-from` as my sendmail_command and I can send a mail just fine if a set the to field to `"Luc, der Hoffmann" <my-vaid-email@inter.net>`. I get this error popup in alot though:
```
Can not find key for '"Luc'.
Can not find key for 'my-vaid-email@inter.net'.
```
but I assume that is due to the autoencrypt option. To me this shows that we indeed do something undesirable to the to field but also that the to field is not passed to the sendmail_command (or is it just my msmtp --read-recipients that does not care about the arguments given?)

What kind of delivery errors do you get exactly?

**Comment by pazz** [comment]
Sorry, the initial statement wasn't too clear.
The point is that `"X X x@x.com, Y Y" y@y.com` has a different semantics than what I intended (and typed). This describes a single recipient with address ` y@y.com` and name `X X x@x.com, Y Y`.

**Comment by lucc** [comment]
Yes I get the result that you intend (one recipient with a strange name). What do you get?

**Comment by pazz** [comment]
I get the same. but what I intended (and typed) was two recipients!

**Comment by pazz** [comment]
fixed in #993
